build.gradle

```diff
@@ -1,24 +1,24 @@
 buildscript {
-  ext.kotlin_version = '1.0.2'
+  ext.kotlin_version = '1.0.6'
   repositories { jcenter() }
-  dependencies { classpath 'com.netflix.nebula:gradle-rxjava-project-plugin:2.+',
+  dependencies { classpath 'com.netflix.nebula:gradle-rxjava-project-plugin:4.+',
                            "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version" }
 }
 
-apply plugin: 'rxjava-project'
+apply plugin: 'nebula.rxjava-project'
 apply plugin: 'kotlin'
 
 dependencies {
-    compile 'io.reactivex:rxjava:1.1.5'
+    compile 'io.reactivex:rxjava:1.2.7'
     compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    testCompile 'org.funktionale:funktionale:0.8'
+    testCompile 'org.funktionale:funktionale:0.9'
```

Comment:
This one could be replaced by `org.funktionale:funktionale-partials:1.0.0-final`